test/spmd/test_spmd_debugging.py

```diff
@@ -255,6 +267,8 @@ def test_single_host_partial_replication_cpu(self):
     from torch_xla.distributed.spmd.debugging import visualize_tensor_sharding
     device = xm.xla_device()
     num_devices = self.n_devices
+    if num_devices != 1:
```

Comment:
Naive question, but is there a case where `PJRT_DEVICE=CPU` has n_devices > 1?

Reply:
thanks, Jon, very good question, checked with TPU Pod:
```
$ gcloud alpha compute tpus tpu-vm ssh ${TPU_NAME}  --project ${PROJECT_ID} --zone  ${ZONE} --worker all --command='PJRT_DEVICE=CPU python3 -c "import torch; import torch_xla; import torch_xla.runtime as xr; print(xr.global_runtime_device_count());"'
Using ssh batch size of 4. Attempting to SSH into 1 nodes with a total of 4 workers.
SSH: Attempting to connect to worker 0...
SSH: Attempting to connect to worker 1...
SSH: Attempting to connect to worker 2...
SSH: Attempting to connect to worker 3...
1
1
1
1
```
looks like `PJRT_DEVICE=CPU` only has n_devices = 1